main/views.py

```diff
@@ -1,3 +1,6 @@
 from django.shortcuts import render
 
 # Create your views here.
+
+def index(request):
+    return render(request, 'main\index.html')
```

Comment:
Since you imported the urls from main into the urls.py under RateMyLandlord, you don't have to specify the path to index.html. You can just pass 'index.html' to render.